Pod/Classes/URBNAlertAction.m

```diff
@@ -25,7 +26,14 @@ + (URBNAlertAction *)actionWithTitle:(NSString *)title actionType:(URBNAlertActi
 }
 
 - (BOOL)isButton {
-    return (self.actionType == URBNAlertActionTypeNormal || self.actionType == URBNAlertActionTypeDestructive);
+    return (self.actionType == URBNAlertActionTypeNormal || self.actionType == URBNAlertActionTypeDestructive || self.actionType == URBNAlertActionTypeCancel);
```

Comment:
This totally works, but would it be better/easier to just `return self.actionType != URBNAlertActionTypePassive;`